ssn/chapters/Introduction.html

```diff
@@ -2,61 +2,77 @@
 <h2>Introduction</h2>
 
 <p>
-  Sensors are a major source of data available on the Web today. However, searching, reusing, integrating, and
-  interpreting sensor data requires more than just the sensing result.
-  For proper interpretation of these values, knowledge of the feature of interest, such as a river, the observed property, such as flow velocity, the sampling strategy, such as the specific locations and times at which the velocity was measured, and a variety of other information are required.
+  Sensors are a major source of data available on the Web today.
+  However, searching, reusing, integrating, and interpreting sensor data requires more than just the sensing result.
+  For proper interpretation of these values, knowledge of the feature of interest, such as a river, the observed
+  property, such as flow velocity, the sampling strategy, such as the specific locations and times at which the velocity
+  was measured, and a variety of other information are required.
 </p>
 <p>
-  In addition, the rise of the Web of Things makes actuators and the data they produce first-class citizens of the Web. 
-  Parallel to sensors and observations, searching, reusing, integrating, and interpreting actuator data requires more 
+  Furthermore, the Web of Things makes actuators and the data they produce first-class citizens of the Web.
+  Parallel to sensors and observations, searching, reusing, integrating, and interpreting actuator data requires more
   than just the actuation input and result values.
-  Knowledge of the feature of interest, such as a smart thermostat, the actuated property, such as temperature 
+  Knowledge of the feature of interest, such as a smart thermostat, the actuated property, such as temperature
   setpoint, and a variety of other information are required.
 </p>
 <p>
   OGC's Sensor Web Enablement standards [[SWE]], [[SensorML]], [[OandM]], [[OMS]], [[STA]] provide
-  conceptual models, as well as some encodings for the annotation of sensors and their observations.
+  conceptual models, and some encodings for the annotation of sensors and their observations.
```

Comment:
```suggestion
  conceptual models and some encodings for the annotation of sensors and their observations.
```

Reply:
fixed in https://github.com/w3c/sdw-sosa-ssn/pull/341/commits/b96c703ff0597d9e3057f34629e7da59f896c544